UnityProject/Assets/Scripts/Messages/Client/Admin/RequestRespawnPlayer.cs

```diff
@@ -24,7 +24,9 @@ void VerifyAdminStatus()
 			if (deadPlayer == null || deadPlayer.Script == null) return;
```

Comment:
deadPlayer.Script is already checked here :p

Reply:
one step ahead of you, already fixed it 😎